app/models/report.rb

```diff
@@ -1,7 +1,43 @@
 # frozen_string_literal: true
 
 class Report < ApplicationRecord
+  after_commit :push_to_slack, on: :create
+
   belongs_to :user
 
   validates :task_today, :obstacles_today, presence: true
+
+  def push_to_slack
+    thread_ts = SlackThread.find_by(created_on: Time.zone.today).thread_ts
+
+    uri = URI.parse('https://slack.com/api/chat.postMessage')
+
+    req_options = { use_ssl: uri.scheme == 'https' }
+
+    Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
+      http.request(set_request_body(uri, thread_ts))
+    end
+  end
+
+  private
+
+  def set_request_body(uri, thread_ts)
+    request = Net::HTTP::Post.new(uri)
+    request.content_type = 'application/json'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/reportedly-api/blob/fb89d1c90ff855d229a9544a010555f929601d1c/app/models/report.rb#L26"></span>Refers to 'request' more than self (maybe move it to another class?)</td>
    </tr>
  </tbody>
</table>